scw_serverless/config/function.py

```diff
@@ -74,19 +58,18 @@ class Function:
 
     name: str
     handler: str  # Path to the handler
-    runtime: sdk.FunctionRuntime
     environment_variables: Optional[dict[str, str]] = None
     min_scale: Optional[int] = None
     max_scale: Optional[int] = None
     memory_limit: Optional[int] = None
     timeout: Optional[str] = None
-    secret_environment_variables: Optional[list[sdk.Secret]] = None
-    privacy: Optional[sdk.FunctionPrivacy] = None
+    secret_environment_variables: Optional[dict[str, str]] = None
+    privacy: Privacy = "public"
     description: Optional[str] = None
-    http_option: Optional[sdk.FunctionHttpOption] = None
```

Comment:
Those types were changed to avoid needing to import the SDK in the config package. The config package is used by the decorators, and will be imported passively when running the functions on Scaleway. With those changes, it will be possible to vendor `scw-serverless` with none of its dependencies when deploying to scaleway Functions.

This is good because the next step is to add the dev command and therefore add Flask (transitively) as a dependency.